packages/typespec-python/test/generated/azure-core-lro-standard/_specs_/azure/core/lro/standard/_operations/_operations.py

```diff
@@ -457,11 +461,15 @@ def _export_initial(  # pylint: disable=inconsistent-return-statements
         response_headers = {}
         response_headers["Operation-Location"] = self._deserialize("str", response.headers.get("Operation-Location"))
 
+        deserialized = _deserialize(_models.ExportedUser, response.json().get("result"))
```

Comment:
According to [guideline about Long-running Action Operations](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-action-operations), final result is in "result" filed so we need to extract it before deserialization:
![image](https://github.com/Azure/autorest.python/assets/70930885/9ebc79ff-04c7-497f-89f0-1bb9318465ca)